packages/main/src/SliderBase.ts

```diff
@@ -28,11 +28,8 @@ type DirectionStart = "left" | "right";
  *
  * <h3 class="comment-api-title">Overview</h3>
  *
- *
  * @constructor
- * @author SAP SE
- * @alias sap.ui.webc.main.SliderBase
- * @extends sap.ui.webc.base.UI5Element
+ * @extends UI5Element
  * @public
  */
 @customElement({
```

Comment:
move this JSDoc to the class declaration and create 2 static getters in the class respectively for styles and renderer. SliderBase is class, not a custom element so the decorator should not be used here

Reply:
Done